Modules/@babylonjs/react-native/windows/BabylonReactNative/EngineViewManager.h

```diff
@@ -0,0 +1,73 @@
+#pragma once
+#include "NativeModules.h"
+#include "winrt/Microsoft.ReactNative.h"
+#include "JSI/jsi.h"
+#include <unordered_set>
+
+namespace winrt::BabylonReactNative::implementation {
+
+    struct EngineViewManager
+        : winrt::implements<
+        EngineViewManager,
+        winrt::Microsoft::ReactNative::IViewManager,
+        winrt::Microsoft::ReactNative::IViewManagerWithReactContext,
+        winrt::Microsoft::ReactNative::IViewManagerWithNativeProperties,
+        winrt::Microsoft::ReactNative::IViewManagerWithExportedEventTypeConstants,
+        winrt::Microsoft::ReactNative::IViewManagerWithCommands>
+        , std::enable_shared_from_this<EngineViewManager> {

```

Comment:
Don't combine `winrt::implements` with `std::enable_shared_from_this` since WinRT has its own reference counting mechanism that the STL's will conflict with. You should still be able to get weak pointers from C++/WinRT using `get_weak()`.

Reply:
looks like we were never using the standard library logic and this is a stale inheritance